tests/yam/validate/validate_user.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate alp user with password
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+
+sub run {
+    my $user = "test";
+    select_serial_terminal();
+    assert_script_run "useradd -m $user";
+    assert_script_run "echo '$user:$testapi::password' | chpasswd";
+    type_string("su - $user", lf => 1);
+
+    # Validate new alp user with password
+    $user = "bernhard";
```

Comment:
how about `$testapi::user`? :)

Reply:
Thanks for comments, current code will be recontructed later.